<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LE-PF1EPTKS" name="com.automation.qa.testcases.LoginPageTest" tests="4" failures="1" timestamp="09 Mar 2020 15:53:20 GMT" time="61.838" errors="0">
  <testcase name="verifyPageTitleTest" time="14.615" classname="com.automation.qa.testcases.LoginPageTest"/>
  <testcase name="verifyLogoDisplayTest" time="13.770" classname="com.automation.qa.testcases.LoginPageTest"/>
  <testcase name="verifyLoginTest" time="16.604" classname="com.automation.qa.testcases.LoginPageTest"/>
  <testcase name="verifyInvalidLoginAndCaptureScreenshotTest" time="16.849" classname="com.automation.qa.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message=" Unsuccessful Login please check username and password.Screenshot captured for debugging purpose. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError:  Unsuccessful Login please check username and password.Screenshot captured for debugging purpose. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at com.automation.qa.testcases.LoginPageTest.verifyInvalidLoginAndCaptureScreenshotTest(LoginPageTest.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyInvalidLoginAndCaptureScreenshotTest -->
</testsuite> <!-- com.automation.qa.testcases.LoginPageTest -->
